README.md

```diff
@@ -57,7 +57,7 @@ If you find a bug, please report an issue on GitHub.
 
 # Copyright and license
 
-The snowplow-web package is Copyright 2021 Snowplow Analytics Ltd.
+The snowplow-web package is Copyright 2022 Snowplow Analytics Ltd.
```

Comment:
I think this should be 2021-2022

Reply:
Thanks for the heads up, should be all good now!